src/behaviours/preferences/navigation-to-extension-specific-preferences.test.tsx

```diff
@@ -48,7 +48,7 @@ describe("preferences - navigation to extension specific preferences", () => {
         const getRendererExtensionFake = getRendererExtensionFakeFor(applicationBuilder);
         const testExtension = getRendererExtensionFake(extensionStubWithExtensionSpecificPreferenceItems);
 
-        applicationBuilder.addExtensions(testExtension);
+        applicationBuilder.extensions.renderer.enable(testExtension);
```

Comment:
why is this one not awaited and the rest are?

Reply:
Hm, good catch. It's because in upcoming PR there is change to remove async from enabling extension for good. However, in here `await` is correct.

Reply:
Fixed